Makefile

```diff
@@ -46,7 +46,7 @@ lint: lint-flake8 lint-pylint
 
 # E501 and F401 are ignored because Pylint performs similar checks.
 lint-flake8:
-	flake8 . --ignore E501,F401 --exclude docs/_build,build
+	flake8 . --ignore E501,F401,W504 --exclude docs/_build,build
```

Comment:
Can we add a comment on why we are ignoring W504?
Like 
```
# Ignoring W504 since pep8 requires line breaks after the binary operator.
```

Reply:
This comment just needs to be updated for W504 since F401 is no longer ignored or update per @ragabala

Reply:
I will update with a comment, but a better/long term solution will be fix all those errors.

Reply:
> I will update with a comment, but a better/long term solution will be fix all those errors.

So, write-up and issue and assign it to the new guy? It would be good for him to get familiar with linting and getting some commits in anyway, right?